lib/vsc/utils/fancylogger.py

```diff
@@ -431,7 +440,7 @@ def getRootLoggerName():
         return "not available in optimized mode"
 
 
-def logToScreen(enable=True, handler=None, name=None, stdout=False):
+def logToScreen(enable=True, handler=None, name=None, stdout=False, colorize='never'):
```

Comment:
> > -def logToScreen(enable=True, handler=None, name=None, stdout=False):
> > +def logToScreen(enable=True, handler=None, name=None, stdout=False, colorize='never'):
>
> does it make sense to use `None` here as default, and then fall back to using `'never'` in `_screenLogFormatterFactory` if it's `None`?

why? what advantages/differences would it have compared to defaulting to ``'never'``?